packages/vscode-extension/package.json

```
@@ -620,6 +629,7 @@
     "format": "prettier --write --list-different src",
     "build:tests": "tsc --project tsconfig.test.json",
     "build:plugins": "bash ./scripts/install-dev-plugins.sh",
+    "build:icons": "npm exec fantasticon -- -c ./font-icons/fantasticon.config.json",
```

Since we have `fantasticon` as dev dependency, shouldn't
```suggestion
    "build:icons": "fantasticon -c ./font-icons/fantasticon.config.json",
```
be sufficient?

---

You're right. This works. I just blindly copied command I was testing from command line